x/auth/pubkey/types/constants.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+const (
+	// ModuleName defines the module's name.
+	ModuleName = "pubkey"
```

Comment:
Could we name this module changepubkey or something like that? I feel just calling it pubkey might be confusing.